docs/build-aea-programmatically.md

```diff
@@ -71,6 +73,40 @@ Next, we add the echo skill which will bounce our messages back to us. We first
     builder.add_skill("./packages/fetchai/skills/echo")
 ```
 
+Also, we can add a component that was instantiated programmatically. :
+``` python
+    # create skill and handler manually
+    from aea.protocols.base import Message
+    from aea.protocols.default.message import DefaultMessage
+    from aea.skills.base import Handler
+
+    class DummyHandler(Handler):
+        """Dummy handler to handle messages."""
+
+        SUPPORTED_PROTOCOL = DefaultMessage.protocol_id
+
+        def setup(self) -> None:
+            """Noop setup."""
```

Comment:
What does `noop` mean?

Reply:
I think is an abbreviation for [no operation](https://en.wikipedia.org/wiki/NOP_(code)).

Reply:
yes, it is. it is. it's empty, and nothing to write in comment.